src/main/java/org/folio/service/manager/export/ExportManagerImpl.java

```diff
@@ -120,7 +121,7 @@ private SrsLoadResult loadSrsMarcRecordsInPartitions(List<String> identifiers, O
   private List<JsonObject> loadInventoryInstancesInPartitions(List<String> singleInstanceIdentifiers, OkapiConnectionParams params) {
     List<JsonObject> instances = new ArrayList<>();
     Lists.partition(singleInstanceIdentifiers, INVENTORY_LOAD_PARTITION_SIZE).forEach(partition -> {
-        List<JsonObject> partitionLoadResult = recordLoaderService.loadInventoryInstancesBlocking(partition, params);
+        List<JsonObject> partitionLoadResult = recordLoaderService.loadInventoryInstancesBlocking(partition, params, INVENTORY_LOAD_PARTITION_SIZE);
```

Comment:
@andrii-novytskyi  Have you verified that inventory API is able to support 10 ids at a time or more?

Reply:
@KVupp yep, here is the screen where i verified it with 11 uuids:
![image](https://user-images.githubusercontent.com/60778653/78373737-a7ae7900-75d3-11ea-8b5b-be3af560c87a.png)

Reply:
Can we add the maximum number as the partition then just to reduce number of calls to inventory?

Reply:
Just want to mention that in this case we also need to increase "limit" request parameter respectively

Reply:
@KVupp i tested with 20 numbers for SRS and inventory storage, and increased limit to 20.